instrument.yml

```diff
@@ -0,0 +1,11 @@
+name: Honeydew
+run: onEach
+results:
+  - dir: ${instrument}/results
+    files:
+      - "*.json"
+commands:
+  - name: run honeydew
+    win: honeydew.exe extract ${repo} results/model.json
```

Comment:
The command was simplified, as I understand from @andrei2699. It should only be "Honeydew extract ${repo}"